lib/testing/Testing.js

```diff
@@ -1,11 +1,14 @@
 "use strict";
 
 var Configure = require("../application/Configure");
+var Layout = require("../viewing/Layout");
+var noop = require("../util/noop");
 
 var Testing = {
 
     setup: function() {
         Configure.backbone();
+        Layout.prototype.reattach = noop;
```

Comment:
when layout reattaches during jasmine tests, it takes over the test runner html and breaks the suite. this line allows client renderer to run safely